docs/getting-started.md

```diff
@@ -1,27 +1,47 @@
 # Getting Started with Scalar DL
 
-This document explains how to get started with Scalar DL by running your first simple contract with the client SDK.
-Here, we assume that you already have a certificate and a private key required to run contracts.
+This document explains how to get started with Scalar DL by running your first simple contract with the Client SDK.
+Here, we assume that you have already installed Scalar DL, and Scalar DL Ledger is running and listening to 50051 and 50052 ports on localhost. If you don't have such an environment, please follow [the document](./installation-with-docker.md). We also assume that you already have a certificate and a private key required to run contracts.
 
 ## What is Scalar DL?
 
-Scalar DL is a blockchain-inspired distributed ledger platform, which achieves:
+Scalar DL is a scalable and practical Byzantine fault detection middleware for transactional database systems that achieves correctness, scalability, and database agnosticism.
 
-* High tamper-evidence of data
-* High scalablility and availablity
-* Linearizable consistency and ACID
+Scalar DL is composed of Ledger, Auditor, and Client SDK as shown in the following figure. Scalar DL Ledger manages application data in its own unique way using hash-chain and digital signature. Scalar DL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data.
+The Client SDK is a set of user-facing programs to interact with Ledger and Auditor. For more details, please read [the design doc](design.md) and [the implementation details](implementation.md).
 
-It manages data as a set of assets, where an asset is composed of a history of data identified by a key called `asset_id` and a historical version number called `age`.
-In this document, you will create a very simple application to manage an asset's status.
+<p align="center">
+<img src="https://github.com/scalar-labs/scalardl/raw/master/docs/images/scalardl.png" width="480" />
+</p>
+
+Scalar DL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of a history of a record identified by a key called `asset_id` and a historical version number called `age`.
+In this document, you will create a very simple application to manage an asset's status using Scalar DL Client SDK.
+
+## Download the Client SDK
+
+The Client SDK library is available on [Maven Central](https://search.maven.org/search?q=a:scalardl-java-client-sdk). You can install it in your application using your build tool such as Gradle.
+For example in Gradle, you can add the following dependency to your build.gradle.
+
+```
+dependencies {
+    compile group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '<version>'
+}
+```
+
+From here, let's use the [scalardl-java-client-sdk]() repo that has a sample `build.gradle`, sample contracts, and tools for quick testing.
```

Comment:
Should we have https://github.com/scalar-labs/scalardl-java-client-sdk here?

Reply:
Thank you!